Classifier_Klasse.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun  7 13:55:25 2021
+
+@author: Natasha
+"""
+
+import pandas as pd
+from feature_class_punctuation import PunctFeatures
+from Word_features import count_stopwords
+
+class Classifier():
```

Comment:
:beetle: Im Moment übergibst du predict, das trainierte Model aus train_model und wendest dieses an um Klassen für die Daten in self.df vorherzusagen. Wenn wir später den Classifier auf Instanzen anwenden wollen, auf denen wir nicht trainiert haben, dann wäre es vielleicht sinnvoller, wenn du das trainierte Model nicht zurückgeben würdest, sondern in einer Instanzvariable speicherst und stattdessen der predict Funktion einen DataFrame übergibst mit den Instanzen, für die eine Klasse ermittelt werden soll.